forest/sycamore/tests/test_functions.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import os
+
+from forest.sycamore.survey_config import get_all_interventions_dict
+from forest.sycamore.functions import aggregate_surveys
+from forest.sycamore.survey_config import gen_survey_schedule
+
+# Needed so that relative paths in the below file will work
+os.chdir(os.path.join("forest", "sycamore", "tests"))
```

Comment:
I am not sure why this is necessary.

Reply:
I need to look more into how relative paths work in pytest myself. But, when I did not include this line, pushing to GitHub would return an error.

Reply:
But not when running the tests locally? What was the error?

Reply:
You can see it [here](https://github.com/onnela-lab/forest/runs/5348311431?check_suite_focus=true)

> forest/sycamore/tests/test_functions.py:7: in <module>
> [16](https://github.com/onnela-lab/forest/runs/5348311431?check_suite_focus=true#step:9:16)
>     assert get_all_interventions_dict("empty_intervention_data.json") == {}
> [17](https://github.com/onnela-lab/forest/runs/5348311431?check_suite_focus=true#step:9:17)
> forest/sycamore/survey_config.py:4[18](https://github.com/onnela-lab/forest/runs/5348311431?check_suite_focus=true#step:9:18): in get_all_interventions_dict
> 18
>     full_dict = read_json(filepath)
> [19](https://github.com/onnela-lab/forest/runs/5348311431?check_suite_focus=true#step:9:19)
> forest/sycamore/functions.py:28: in read_json
> [20](https://github.com/onnela-lab/forest/runs/5348311431?check_suite_focus=true#step:9:20)
>     with open(study_dir, 'r') as f:
> [21](https://github.com/onnela-lab/forest/runs/5348311431?check_suite_focus=true#step:9:21)
> E   FileNotFoundError: [Errno 2] No such file or directory: 'empty_intervention_data.json'

Reply:
This is because you've used relative paths to JSON files. They will work only if you run `pytest` from the `tests` dir. The standard way to solve this is something like:
```
TEST_DATA_DIR = os.path.dirname(os.path.abspath(__file__))
```
then:
```
get_all_interventions_dict(os.path.join(TEST_DATA_DIR, "empty_intervention_data.json")
```